tutorial/versioning/versions.rst

```diff
@@ -0,0 +1,219 @@
+.. _tutorial_versioning_versions:
+
+Versions
+========
+
+Let's start with a very simple recipe:
```

Comment:
Maybe we can do a summary of what we are going to cover in this section before beginning the example?

Reply:
Maybe the structure could be like...

Versions
======

Write a little summary... that we will explain how to put versions inrecipes and after that to automating versions and then what comes next, etc.

Versions in recipes
-------------------

.....

Automating versions
-------------------
.....
.....